scripts/download_codeforces.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# Run script from code2seq dir using command:
+#    sh scripts/download_data.sh
+TRAIN_SPLIT_PART=$1
+VAL_SPLIT_PART=$2
+TEST_SPLIT_PART=$3
+DEV=$4
+SHUFFLE=$5
+DATA_DIR=./data
+DATASET_NAME=cf
+ASTMINER_PATH=../astminer/build/shadow/lib-0.5.jar
+SPLIT_SCRIPT=./scripts/split_dataset.sh
```

Comment:
These are similar to other args in `download_data.sh`, so you can pass them like previous args.